src/openforms/submissions/tests/test_tasks_pdf.py

```diff
@@ -23,3 +23,6 @@ def test_submission_report_metadata(self):
         self.assertEqual(submission, report.submission)
         self.assertTrue(report.content.name.endswith("Test_Form.pdf"))
         self.assertEqual("some-id", report.task_id)
+
+    def test_submission_report_multivalue(self):
+        pass
```

Comment:
Can you delete this empty test?

Reply:
Yes, it was cruft from 'smart' branch switching